services/src/test/java/org/broadleaf/payment/service/gateway/PayPalCheckoutTransactionServiceIT.java

```diff
@@ -0,0 +1,546 @@
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalCaptureRequest;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalCaptureResponse;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalCaptureRetrievalRequest;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalCaptureRetrievalResponse;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalRefundRequest;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalRefundResponse;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalSaleRequest;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalSaleResponse;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalSaleRetrievalRequest;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalSaleRetrievalResponse;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalVoidRequest;
+import org.broadleafcommerce.vendor.paypal.service.payment.PayPalVoidResponse;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import com.broadleafcommerce.money.util.MonetaryUtils;
+import com.broadleafcommerce.paymentgateway.domain.PaymentRequest;
+import com.broadleafcommerce.paymentgateway.domain.PaymentResponse;
+import com.broadleafcommerce.paymentgateway.domain.enums.DefaultTransactionFailureTypes;
+import com.broadleafcommerce.paymentgateway.domain.enums.DefaultTransactionTypes;
+import com.broadleafcommerce.paymentgateway.service.exception.PaymentException;
+import com.paypal.api.payments.Authorization;
+import com.paypal.api.payments.Capture;
+import com.paypal.api.payments.DetailedRefund;
+import com.paypal.api.payments.Error;
+import com.paypal.api.payments.Payment;
+import com.paypal.api.payments.RelatedResources;
+import com.paypal.api.payments.Sale;
+import com.paypal.api.payments.Transaction;
+import com.paypal.base.rest.PayPalRESTException;
+
+import java.util.Collections;
+
+/**
+ * @author Dima Myroniuk (dmyroniuk)
+ */
+@SpringBootTest
+public class PayPalCheckoutTransactionServiceIT {
+
+    private static final String CUSTOM_ID = "customId";
+    private static final String RAW_RESPONSE = "rawResponse";
+    private static final String AUTH_ID = "authId";
+    private static final String REASON_CODE = "reasonCode";
+    private static final String PAYMENT_ID = "paymentId";
+    private static final String PAYER_ID = "payerId";
+    private static final String SALE_ID = "saleId";
+    private static final String BILLING_AGREEMENT_ID = "billingAgreementId";
+    private static final String CAPTURE_ID = "captureId";
+    private static final String VOIDED_AUTHORIZATION_ID = "voidedAuthorizationId";
+    private static final String DETAIL_REFUND_ID = "detailRefundId";
```

Comment:
These should be moved to `MessageConstants.java`

Reply:
These fields are required only for the tests. I don't think `MessageConstants.java` should contain these constants.